tasks_nodejs_basics/task_3.ts

```diff
@@ -0,0 +1,61 @@
+//npm run start:dev tasks_nodejs_basics/task_3.ts
+//D:\test\list.json
+
+import * as fs from 'fs';
```

Comment:
`node:fs/promise`
`node:path`
etc

Reply:
I also rewrited code for task_3 when I have changed fs to node:fs/promises